.devcontainer/Dockerfile.backend

```diff
@@ -21,6 +21,5 @@ EOF
 
 
 WORKDIR /mzai
-ENV PYTHONPATH="/mzai"
-CMD zip lumigator/python/mzai/backend/api/deployments lumigator/python/mzai/backend/api/deployments.zip
```

Comment:
we still need to do this until we remove Ray Serve, I think

Reply:
Tested and working without this.